### PR TITLE
issue3894: update vagrant sections of contributing and readme doc to …

### PR DESCRIPTION
**Pull request by gabemontero**
…account for recent tweaks in runtime

@bparees pending a last minute clarification on whether or not to include --volume-dir in the start example provided, here are the doc changes for dealing with the recent deltas since walking through these steps last month.

Fixed https://github.com/openshift/origin/issues/3894

### CODE REVIEW

File: examples/sample-app/README.md [hunk 1]
```diff
@@ -41,8 +41,9 @@ Inside of your Vagrant instance, the path to the origin directory is `/data/src/
 
 	$ cd /data/src/github.com/openshift/origin
 
-Run an advance build of the OpenShift binaries before continuing:
+Run an advance build of the OpenShift binaries before continuing, where since a OpenShift compile is occurring, you need to override the default amount of memory assigned to the VM:
 
+	$ export OPENSHIFT_MEMORY=2096
```
Comment (bparees) [inline examples/sample-app/README.md]: wouldn't this need to be set prior to vagrant up?

Comment (gabemontero) [inline examples/sample-app/README.md]: ah ... that's right - I had it the correct way in CONTRIBUTING.adoc as well.

**Comment by gabemontero** [comment]
@bparees Order of setting of env var in README.md addressed

**Comment by bparees** [comment]
lgtm [merge]

**Comment by soltysh** [comment]
@gabemontero thanks for writing that, I thought I commented on the related issue but my comment apparently wasn't sent.

**Comment by gabemontero** [comment]
@soltysh No problem - thanks for following up!

**Pull request merged**